cypress/integration/08-save_and_start_test_spec.js

```diff
@@ -100,7 +100,8 @@ describe("add patient and save and start test", () => {
     cy.visit("/");
     cy.get(".usa-nav-container");
     cy.get("#desktop-patient-nav-link").click();
-    cy.get("#add-patient-button").click();
+    cy.get("#add-patient").click();
+    cy.get("#individual_add-patient").click();
```

Comment:
Do you think we should add a new test in one of these files for the bulk patient upload page?

Reply:
Yes I'll be working on it!